src/ContractInteract/UtilityToken.js

```diff
@@ -0,0 +1,346 @@
+/**
+ * @typedef {Object} UtilityTokenConfig
+ *
+ * @property {string} deployer Address to be used to send deployment transactions.
+ * @property {string} valueToken Address of EIP20 Token on Origin chain.
+ * @param {string} symbol Symbol of utility token.
+ * @param {string} name Name of utility token.
+ * @param {string} decimal Decimal of utility token.
+ * @property {string} organization Address of Organization contract managing
+ *                                 Utility token.
+ */
+
+'use strict';
+
+const BN = require('bn.js');
+const Web3 = require('web3');
+
+const AbiBinProvider = require('../AbiBinProvider');
+const Contracts = require('../Contracts');
+const Utils = require('../utils/Utils');
+const { validateConfigKeyExists } = require('./validation');
+
+const ContractName = 'UtilityToken';
+
+/**
+ * Contract interact for UtilityToken contract.
+ */
+class UtilityToken {
+  /**
+   * Constructor for UtilityToken.
+   *
+   * @param {Object} web3 Web3 object.
+   * @param {string} address UtilityToken contract address.
+   */
+  constructor(web3, address) {
+    if (!(web3 instanceof Web3)) {
+      throw new TypeError("Mandatory Parameter 'web3' is missing or invalid");
+    }
+    if (!Web3.utils.isAddress(address)) {
+      throw new TypeError(
+        `Mandatory Parameter 'address' is missing or invalid: ${address}`,
+      );
+    }
+
+    this.web3 = web3;
+    this.address = address;
+
+    this.contract = Contracts.getUtilityToken(this.web3, this.address);
+
+    if (!this.contract) {
+      throw new Error(`Could not load Utility contract for: ${this.address}`);
+    }
+
+    this.approve = this.approve.bind(this);
+    this.approveRawTx = this.approveRawTx.bind(this);
+    this.allowance = this.allowance.bind(this);
+    this.isAmountApproved = this.isAmountApproved.bind(this);
+    this.balanceOf = this.balanceOf.bind(this);
+    this.setCoGateway = this.setCoGateway.bind(this);
+    this.setCoGatewayRawTx = this.setCoGatewayRawTx.bind(this);
+  }
+
+  /**
+   * Setup for Utility token contract. Deploys the contract and initializes
```

Comment:
Setup for `UtilityToken` doesn't initialize anything. That's only applicable for `OSTPrime`.